flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1408,5 +1408,10 @@
     },
     "org.srb2.SRB2Kart": {
         "flathub-json-modified-publish-delay": "extra-data"
+    },
+    "com.riverbankcomputing.PyQt.BaseApp": {
+        "flathub-json-modified-publish-delay": "This is a BaseApp",
+        "flathub-json-skip-appstream-check": "This is a BaseApp",
+        "finish-args-not-defined": "This is a BaseApp"
```

Comment:
I guess we should handle these exceptions in the linter code for all BaseApps. I will look into it.

Reply:
https://github.com/flathub/flatpak-builder-lint/commit/827794d0afc96afacaf2cbe4c1b136b7fcc36379 and https://github.com/flathub/flatpak-builder-lint/commit/4bfe5d874e1afdb93c69959e64c9b7226345de92 fix it.

Reply:
this was done later in https://github.com/flathub-infra/flatpak-builder-lint/commit/d47ac5c881f42eb34416006eb4c8a1d43362edf2 and https://github.com/flathub-infra/flatpak-builder-lint/commit/3efa25af3229ee2b89a7e8b7becd213e79ebd05e